<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountManageDialog</class>
 <widget class="QWidget" name="AccountManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理账号</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editPassBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editAdminPassBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(99, 193, 255);</string>
   </property>
   <property name="text">
    <string>修改管理员密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
